dallinger/command_line.py

```diff
@@ -318,6 +319,20 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1, exp_config=
     (id, tmp) = setup_experiment(debug=False, verbose=verbose, app=app,
                                  exp_config=exp_config)
 
+    # Register the experiment on OSF
+    # Load configuration.
+    config = get_config()
+    if not config.ready:
+        config.load_config()
+
+    try:
+        config.get('osf_access_token')
```

Comment:
This doesn't actually do anything. If the token hasn't been set, it returns None.